folly/portability/Config.h

```diff
@@ -37,3 +37,7 @@
 #include <AvailabilityMacros.h> // @manual
 #include <TargetConditionals.h> // @manual
 #endif
+
+#ifdef __FreeBSD__
+#include <sys/types.h>
```

Comment:
What is this include for?

Reply:
pid_t/fork_t

Reply:
Let's move this include to those files which need it.